gh-85935: Explicitly document the case nargs=0 in argparse (#125302)
python/cpython: Doc/library/argparse.rst

  .. versionadded:: 3.8

> Only actions that consume command-line arguments (such as ``'store'``,

Reviewer: ```suggestion
Only actions that consume command-line arguments (e.g. as ``'store'``,
```

Author: I think "as" was redundant.
```suggestion
Only actions that consume command-line arguments (e.g. ``'store'``,
```

If the ``nargs`` keyword argument is not provided, the number of arguments consumed
is determined by the action_.  Generally this means a single command-line argument
will be consumed and a single item (not a list) will be produced.

> Actions that do not consume command-line arguments (such as

Reviewer: ```suggestion
Actions that do not consume command-line arguments (e.g.
```
